README.md

```diff
@@ -199,6 +199,28 @@ The default username for kibana dashboard UI will be "elastic" and the password
   echo $PASSWORD
 ```
 
+## Deploying Jaeger on k8s
+
+Jaeger is an opensource end to end distributed system. There is native support for OpenTelemetry in [Jaeger][jaeger].
+
+Before you deploy Jaeger, make sure to follow the section above to "Setup Elasticsearch & FluentD Logging on k8s". 
+
+The elasticsearch instance is required to deploy Jaeger with an ElasticSearch storage backend. 
+
+To deploy the jaeger operator and instance, simply run the [deploy-jaeger-tracing][deploy-jaeger-tracing]
+
+```bash
+  ./deploy-jaeger-tracing.sh
+```
+
+Then to view the jaeger ingress:
+
+```bash
+  kubectl get ingress jaeger-tracing-ingress -n observability
```

Comment:
Would it make sense to use the same monitoring namespace we have for Prometheus? Seems redundant to have a monitoring and an observability namespace.

Reply:
you are right but the problem is jaeger operator has observability namespace harcoded in for the resources : https://github.com/jaegertracing/jaeger-operator/releases/download/v1.35.0/jaeger-operator.yaml

I am not sure but we need to store this in git and update it ourselves-